client/src/js/samples/components/Create/Create.js

```diff
@@ -1,33 +1,40 @@
-import { push } from "connected-react-router";
-import { filter, get, map, replace, split } from "lodash-es";
+import { filter, get, map, replace } from "lodash-es";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['replace' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4748572)